tool/container_tool.py

```diff
@@ -34,23 +34,30 @@ def _execute_command(self,
                        command: list[str],
                        in_container: bool = False) -> sp.CompletedProcess:
```

Comment:
nit: we should just have two separate functions for the "in container" vs not case. It's a bit too confusing to have one function that handles both via a boolean flag.

Reply:
Sure, done